tests/parser.cpp

```diff
@@ -23,25 +23,25 @@ void test(const std::string &input, const std::string &output)
 
 TEST(Parser, builtin_variables)
 {
-  test("kprobe:f { pid }", "Program\n kprobe:f\n  builtin: pid\n");
-  test("kprobe:f { tid }", "Program\n kprobe:f\n  builtin: tid\n");
-  test("kprobe:f { cgroup }", "Program\n kprobe:f\n  builtin: cgroup\n");
-  test("kprobe:f { uid }", "Program\n kprobe:f\n  builtin: uid\n");
-  test("kprobe:f { username }", "Program\n kprobe:f\n  builtin: username\n");
-  test("kprobe:f { gid }", "Program\n kprobe:f\n  builtin: gid\n");
-  test("kprobe:f { nsecs }", "Program\n kprobe:f\n  builtin: nsecs\n");
-  test("kprobe:f { cpu }", "Program\n kprobe:f\n  builtin: cpu\n");
-  test("kprobe:f { curtask }", "Program\n kprobe:f\n  builtin: curtask\n");
-  test("kprobe:f { rand }", "Program\n kprobe:f\n  builtin: rand\n");
-  test("kprobe:f { ctx }", "Program\n kprobe:f\n  builtin: ctx\n");
-  test("kprobe:f { comm }", "Program\n kprobe:f\n  builtin: comm\n");
-  test("kprobe:f { stack }", "Program\n kprobe:f\n  builtin: stack\n");
-  test("kprobe:f { ustack }", "Program\n kprobe:f\n  builtin: ustack\n");
-  test("kprobe:f { arg0 }", "Program\n kprobe:f\n  builtin: arg0\n");
-  test("kprobe:f { retval }", "Program\n kprobe:f\n  builtin: retval\n");
-  test("kprobe:f { func }", "Program\n kprobe:f\n  builtin: func\n");
-  test("kprobe:f { probe }", "Program\n kprobe:f\n  builtin: probe\n");
-  test("kprobe:f { args }", "Program\n kprobe:f\n  builtin: args\n");
+  test("kprobe:f { pid; }", "Program\n kprobe:f\n  builtin: pid\n");
+  test("kprobe:f { tid;}", "Program\n kprobe:f\n  builtin: tid\n");
```

Comment:
```suggestion
  test("kprobe:f { tid; }", "Program\n kprobe:f\n  builtin: tid\n");
```

Reply:
@mmarchini  thanks for reviewing, I'll take a look at the parser/semantic errors on Travis, maybe I missed to update some of the tests.